security-proxy/log4j/log4j.properties

```diff
@@ -25,6 +25,7 @@ log4j.appender.R.layout.ConversionPattern = %d{yyyy-MM-dd HH:mm:ss} %c{1} [%p] %
 # OGC services statistics
 log4j.appender.OGCSTATISTICS=org.georchestra.ogcservstatistics.log4j.OGCServicesAppender
 log4j.appender.OGCSTATISTICS.activated=true
-log4j.appender.OGCSTATISTICS.jdbcURL=jdbc:postgresql://localhost:5432/georchestra
-log4j.appender.OGCSTATISTICS.databaseUser=georchestra
-log4j.appender.OGCSTATISTICS.databasePassword=georchestra
+#The following properties are deprecated, connection pool configured in security-proxy.properties
```

Comment:
How log4j is aware of the username / password credentials, then ?

Reply:
This method of configuration is deprecated but still functional, given ogc-server-statistics is an independent library.
In the context of security-proxy, it sets up a proper connection pool in the datadir's `security-proxy/security-proxy.properties` like in https://github.com/georchestra/datadir/pull/117/files#diff-deabd5a0515a572a41531e11a2dc7830R55